web/src/components/molecules/Asset/Asset/AssetBody/index.tsx

```diff
@@ -43,17 +40,17 @@ const AssetBody: React.FC<Props> = ({
   handleModalCancel,
   handleFullScreen,
 }) => {
-  const { fileName, createdAt, createdBy } = asset;
   const { svgRender, handleCodeSourceClick, handleRenderClick } = useHooks();
-  const formattedCreatedAt = dateTimeFormat(createdAt);
-  const displayUnzipFileList = selectedPreviewType === PreviewType.ZIP;
+  const formattedCreatedAt = dateTimeFormat(asset?.createdAt);
```

Comment:
Is this not getting any TS errors? dateTimeFormat requires the date prop to not be undefined, but asset?.createdAt can be undefined.